### PR TITLE
First code

### PR DESCRIPTION
**Pull request by donice**
In this PR, I
- Made an HTML file, and added the corresponding styles
- Created a ReadMe file
- Created a .gitignore and added the test.md in it

### CODE REVIEW

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @donice  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Good use of github workflows
Great efforts

## Required Changes ♻️
Great efforts so far congrats, only some missing requirements to add and consider:

_Kindly use microverse readme template and customize with your own content also please keep all sections as only FAQ is optional 
_Linters files are missing in your project kindly make sure to follow carefully the steps to set them up correctly , refer to this link [link](https://github.com/microverseinc/linters-config)

_Also please include node_modules to your gitignore file as it will be added to your project after setting up linters so please do this step before linters 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by donice** [comment]
Hello  @souad988 

I fixed my linters and pushed, it worked on my IDE, but I can't find it here. I need help

File: linters.yml
Comment (emmiiorji) [inline linters.yml:1]: Well done for adding this file. However, the linters.yml file is supposed to be in `.github/workflows` directory in order to be recognised by github. Kindly create a folder `.github` and within it, create another folder `workflows` in which you'll place the linters.yml file. Check the [linters configuration](https://github.com/microverseinc/linters-config/tree/master/html-css) for more clarification.

How you'll know that your linters is set up and without errors is that the red cross below will be a green tick:
image

Note: You may run `npx stylelint "**/*.{css,scss}" --fix` to auto fix style errors.

File: README.md
Comment (emmiiorji) [inline README.md:1]: As requested by the previous reviewer, kindly adjust your readme file to follow the [given readme template](https://github.com/microverseinc/readme-template). Remember to customize it.

The readme file gives information about your project and how to use it.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @donice, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<div align='center'>

</div>

 ## Good Highlights✔️

- The test file was well ignored


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,142 @@
...
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```
Comment (stanleySimeon) [inline README.md:141]: Good job making the required changes by the previous reviewer, at this time I suggest making sure this link works and redirects to the MIT license. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @donice, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (donice) [reply README.md:141]: @stanleySimeon  Please can you recheck this

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @donice   :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- App runs well and functions as expected ✅ 
- Professional ReadMe documentation ✅ 
- Good use of Github flow ✅ 
- No bad links ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**